app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,41 @@
  *= require_tree .
  *= require_self
  */
+
+body {
```

Comment:
You might have a nicer time if you can extract this to something like `base.css` — and make `application.css` only responsible for being the place other stuff is required. Not a blocker for this though, as it does seem to be genuinely global styling.